crates/sui-archival/src/lib.rs

```diff
@@ -333,22 +333,27 @@ pub async fn verify_archive_with_genesis_config(
         genesis_committee,
     );
 
-    let mut retries = 0;
-    while retries < num_retries {
-        retries += 1;
-        if let Ok(()) = verify_archive_with_local_store(
+    for i in 0..num_retries {
```

Comment:
should we ensure retries is at least one? I think if retries is 0 this would not execute.

Reply:
I'm not following.. why would it not execute? I'm looping for `num_retries` times unconditionally, but doing an early return from the function on success. So nothing will otherwise stop it from retrying.

In fact, we don't even use the temp var `i`, so I could replace that with `_`

Reply:
Oh understood now, will fix